grl/run_sample_based.py

```diff
@@ -150,19 +152,20 @@ def parse_arguments(return_defaults: bool = False):
 
     final_eval_info, rand_key = eval_episodes(agent, final_network_params, env, rand_key,
                                               n_episodes=args.offline_eval_episodes,
-                                              test_eps=0., action_cond=args.action_cond,
+                                              test_eps=args.offline_eval_epsilon,
                                               max_episode_steps=args.max_episode_steps)
 
     summed_perf = 0
     for ep in final_eval_info['episode_rewards']:
-        summed_perf += sum(ep)
+        full_ep = uncompress_episode_rewards(ep['episode_length'], ep['most_common_reward'], ep['compressed_rewards'])
+        summed_perf += sum(full_ep)
 
     print(f"Final (averaged) greedy evaluation performance: {summed_perf / args.offline_eval_episodes}")
 
     info = {
         'episodes_info': episodes_info,
         'args': vars(args),
-        'final_greedy_eval_rews': final_eval_info['episode_rewards'],
+        'final_eval_rews': final_eval_info['episode_rewards'],
```

Comment:
As a style thing, I'd prefer we consistently write out "rewards" in full, rather than only sometimes

Reply:
Fixed!